fla/models/log_linear_mamba2/modeling_log_linear_mamba2.py

```
@@ -0,0 +1,541 @@
+                self.conv_states[layer_idx].device
+            )
+        return self.conv_states[layer_idx]
+
+    def update_hssm_state(
+        self, layer_idx: int, new_hssm_state: LogLinearAttentionState
+    ) -> LogLinearAttentionState:
+        self.hssm_states[layer_idx] = new_hssm_state
+        return self.hssm_states[layer_idx]
+
+    def reset(self) -> None:
+        for k in self.conv_states.keys():
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)